cores/esp32/HardwareSerial.h

```diff
@@ -149,6 +153,8 @@ typedef enum {
       #define RX1 (gpio_num_t)26
     #elif CONFIG_IDF_TARGET_ESP32S2
       #define RX1 (gpio_num_t)4
+    #elif CONFIG_IDF_TARGET_ESP32C2
+      #define RX1 (gpio_num_t)9
```

Comment:
Pin 9 is already used for I2C. Maybe we can set Serial1 default pins to 10 and 18 (that are free).

```suggestion
    #elif CONFIG_IDF_TARGET_ESP32C2
      #define RX1 (gpio_num_t)10
```